plaso/parsers/sqlite_plugins/windows_eventtranscript.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
+"""SQLite parser plugin for Windows diagnosis EventTranscript database files."""
+import json
+
+from dfdatetime import filetime as dfdatetime_filetime
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+
+class EventTranscriptEventData(events.EventData):
+  """Windows diagnosis EventTranscript event data.
+
+  Attributes:
```

Comment:
Please don't abbreviate things like id, see the style guide for more information. Abbreviation are not always accessible for an international audience.